tasks/pkg-suse.yml

```diff
@@ -0,0 +1,62 @@
+---
+- name: Stat if RPM key already exist
```

Comment:
Could add a condition to do this only on SLES11 and leave ansible handle the RPM key when we don't need the work around ?